utils/publish-snapshot.sh

```diff
@@ -84,7 +84,7 @@ check_for_changes() {
 }
 
 pre_publish_check() {
-    SWCRC=true node -r @swc-node/register "$ROOT/utils/pre-publish-check-ci.ts"
```

Comment:
This started complaining about a @swc/helpers "MODULE_NOT_FOUND". I'm not 100% sure why, but we already use `@swc-node/register` _inside_ this script and it's already `+x` so we can just call it.